go.mod

```diff
@@ -18,9 +18,9 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/google/gofuzz v1.1.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.2.1-0.20201112133003-054c43201b4b
+	github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce
```

Comment:
Let's update it to merge commit `8af0579`.

Reply:
Done.